package/com.unity.formats.usd/Editor/Scripts/Behaviors/UsdMenu.cs

```diff
@@ -23,6 +23,7 @@ namespace Unity.Formats.USD
     public static class UsdMenu
     {
         [MenuItem("USD/Export Selected with Children", true)]
+        [MenuItem("USD/Export Selected with Children as USDA", true)]
```

Comment:
Is it OK that this is defined on Windows but the actual `MenuExportSelectedWithChildrenToUSDA` is not?

Reply:
From first read of the docs it sounded like it was fine but looking at the code it seems like it could be a weird edge case. Although it works as expected on Windows currently I'm not confident it won't break in future. I will change it :)